JavaScript/2-benchmark.js

```diff
@@ -3,15 +3,16 @@
 const benchmark = {};
 module.exports = benchmark;
 
-benchmark.do = (count, retry, tests) => {
+benchmark.do = (count, retry, tests, namesOfTests) => {
```

Comment:
I'd find it more readable to have this variable named `testNames` or something like that.